lifemonitor/auth/templates/auth/apikeys_tab.j2

```diff
@@ -0,0 +1,57 @@
+<div class="m-2 p-2">
+    Allow a user to interact with the LifeMonitor API.
+    An API key acts as a static authentication token and can be generated via the LifeMonitor
```

Comment:
"can be generated via the LifeMonitor..." is redundant in this context, since here we're already in the mentioned place. I'd cut that bit and change this sentence to: "An API key acts as a static authentication token that can be used to quickly try..."

Reply:
Fixed in 6d8edca8b757e8f75b189a28150b98b200d23498